netbox/dcim/tests/test_models.py

```diff
@@ -549,24 +604,27 @@ def test_direct_connection(self):
         self.assertIsNone(endpoint_a.connection_status)
         self.assertIsNone(endpoint_b.connection_status)
 
-    def test_connection_via_single_rear_port(self):
+    def test_connection_via_one_on_one_port(self):
```

Comment:
There is no such thing as a "one-on-one" port. The test method name should describe exactly what's being tested to avoid confusion.